tokio/src/runtime/driver.rs

```diff
@@ -58,6 +61,10 @@ impl Driver {
         ))
     }
 
+    pub(crate) fn is_enabled(&self) -> bool {
```

Comment:
This is added so the scheduler can avoid the extra thread if there are no resources to drive.